tests/testsuite/features_namespaced.rs

```diff
@@ -849,7 +849,8 @@ foo v0.1.0 ([ROOT]/foo)
 bar v1.0.0
 └── bar feature \"default\"
     └── foo v0.1.0 ([ROOT]/foo)
-        └── foo feature \"a\" (command-line)
+        ├── foo feature \"a\" (command-line)
+        └── foo feature \"default\" (command-line)
```

Comment:
While I don't think that this is a bug with this PR per-se I think that this is a bug because neither of the packages here actually have a feature called `default`, so I don't think that this should be printed. I think this could probably be fixed by conditionally inserting the feature `"default"` in `cargo tree` depending on whether the package actually has that feature.

Reply:
Should I open an issue about this?

Reply:
Hm yeah I think it's probably orthogonal-enough from this PR it's best to fix later, and yeah if you wouldn't mind opening an issue that'd be great!

Reply:
Opened #10351